### PR TITLE
First round of minor fixes; Fix #1841

### LINKED ISSUE #1841: Minor repository cleanup
Motivation
----------

Overhaul readmes, organise daemons, etc.

### PR DESCRIPTION
**Pull request by mlassnig**
First round of minor fixes; Fix #1841

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -18,13 +19,7 @@ Accordingly, the `repository <https://github.com/rucio/rucio/>`_  consists of th
...
  * the **hotfix** branch includes the patch for hotfix releases.
 
-Thus, on release day of a feature release both master and next are the same,
-afterwards they diverge until the next feature release.
-Pull requests for **features** are only made against the **next** branch.
-Pull requests for **patches** are made against the **next** and **master** branch, as
-these bugfixes need to be represented in both branches. Thus two
-pull requests are needed for patches, and the helper scripts do it
-automatically for you.
+Thus, on release day of a feature release both master and next are the same, afterwards they diverge until the next feature release. Pull requests for **features** are only made against the **next** branch. Pull requests for **patches** are made against the **next** and **master** branch, as these bugfixes need to be represented in both branches. Thus two pull requests are needed for patches, and the helper scripts do it automatically for you.
```
Comment (cserf) [inline CONTRIBUTING.rst]: You don't mention the hotfix

**Review by cserf — COMMENTED** [review]
Just one small comment

File: CONTRIBUTING.rst (hunk 1 above)
Comment (mlassnig) [reply CONTRIBUTING.rst]: I just removed the useless newlines ;-) I'll add it.

**Review by cserf — DISMISSED** [review]

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -93,30 +86,25 @@ and for a new **feature**::
...
 
 Human Review
 ------------
 
 Anyone is welcome to review merge requests and make comments!
 
-All collaborators, thus the Rucio core development team can approve, request
-changes or close pull requests. Merging of approved pull requests is done by the Rucio
-development lead.
+All collaborators, thus the Rucio core development team can approve, request changes or close pull requests. Merging of approved pull requests is done by the Rucio development lead.
```
Comment (bari12) [inline CONTRIBUTING.rst]: I think we can remove the `All collaborators` part. It's somewhat redundant to Rucio development team. (Maybe ditch the core too)

**Comment by bari12** [comment]
For some reason the tests are not updated here, although they passed. I am force-merging this now.

**Pull request merged**